src/urh/dev/native/lib/bladerf.pyx

```diff
@@ -137,6 +137,19 @@ cpdef bladerf_frequency get_center_freq():
 
     return result
 
+cpdef int set_bias_tee(on_or_off):
+    cdef bool bias_tee = 1 if on_or_off else 0
+    return bladerf_set_bias_tee(_c_device, get_current_bladerf_channel(), bias_tee)
```

Comment:
What will happen here for a ``` BLADERF_VERSION < 2 ```? Will it crash?

Reply:
that's a great question. I only tested for the latest bladerf upstream api version and based on the [README notes api](https://github.com/Nuand/bladeRF/blob/e09200c38e1b273ebe43cd789839e66b0506aead/host/libraries/libbladeRF/doc/doxygen/relnotes_2_0.dox#L631), I set the functions to be called only when `BLADERF_API_VERSION >= 2`, however, when trying to test this with bladerf api `1.9.1-git-fabe4f50` I found out that the methods already exists, for version < 1.9.1 they don't. 

I've adjusted the code to be able to run from 1.9.1 version. If someone enables or disables it using a bladeRF v1, it will throw -8 error code saying that it's not supported for that board. 

Here is the log when enabling it:

```
BladeRF-OPEN: Success
BladeRF: If you experience problems, make sure you place a rbf file matching your device at the correct location. See http://www.nuand.com/fpga_images/ and https://github.com/Nuand/bladeRF/wiki/FPGA-Autoloading
BladeRF-SET_CHANNEL_INDEX to 0: Success
BladeRF-SET_FREQUENCY to 433.92M: Success
BladeRF-SET_SAMPLE_RATE to 2.5M: Success
BladeRF-SET_BANDWIDTH to 2.5M: Success
BladeRF-SET_RF_GAIN to 15: Success
BladeRF-SET_BIAS_TEE_ENABLED to 1: Error Code: -8 (-8)
BladeRF: Initializing BladeRF..
BladeRF: successfully started rx mode
BladeRF: CLOSE:None
```

I made some tweaks in the code so that set/get bias tee can be supported from 1.9.1. 

On top of that, I had to change a little `bladerf_set_frequency` and `bladerf_get_frequency` cause when trying to compile, it was giving me some trouble when trying to convert type variable. Would you mind to test if this is correct? 

gcc version: gcc (GCC) 10.2.1 20201125
```
urh/src/urh/dev/native/lib/bladerf.cpp:4280:163: error: cannot convert ‘__pyx_t_3urh_3dev_6native_3lib_8cbladerf_bladerf_frequency*’ {aka ‘unsigned int*’} to ‘uint64_t*’ {aka ‘long unsigned int*’}
 4280 |   __pyx_v_err = bladerf_get_frequency(__pyx_v_3urh_3dev_6native_3lib_7bladerf__c_device, __pyx_f_3urh_3dev_6native_3lib_7bladerf_get_current_bladerf_channel(0), (&__pyx_v_result));
      |                                                                                                                                                                  ~^~~~~~~~~~~~~~~~
      |                                                                                                                                                                   |
      |                                                                                                                                                                   __pyx_t_3urh_3dev_6native_3lib_8cbladerf_bladerf_frequency* {aka unsigned int*}
```